_episodes/51-managing-software.md

```diff
@@ -375,9 +375,11 @@ Let's create a Project in GitHub to plan the first release of our code.
     or write more detailed comments
     (for that, use the `Convert to issue` option from the `...` menu on the card itself).
 
-   ![Coverting a note to issue](../fig/github-convert-note-to-issue.png){: .image-with-shadow width="1000px"}
+   ![Coverting a task to issue](../fig/github-convert-task-to-issue.png){: .image-with-shadow width="1000px"}
 
-7. You can now drag a card to `In Progress` column to indicate that you are working on it
+7.  In addition to creating new tasks as notes and converting them to issues - you can add an existing 
+issue or pull request (from any repository visible to you) as a task on a column by pasting its URL into the `Add item` field and pressing the `Enter` key.
```

Comment:
should this source-code line be split across 2, rather than being this long?

Reply:
Perhaps something like this:

In addition to creating new tasks as notes and converting them to issues - 
you can add an existing issue or pull request (from any repository visible to you)
as a task on a column by pasting its URL into the `Add item` field 
and pressing the `Enter` key.

Reply:
I thought it was split - will check.

Reply:
Fixed via 44187cf

Reply:
Sorry to be a pain, but could we cut the lines a bit shorter still, and try to align the line breaks with natural breaks in the sentences? It will make future editing of the material easier.

Reply:
How about [01eda56](https://github.com/carpentries-incubator/python-intermediate-development/commit/01eda562dd4b50977968378d4a182faeb14d78e5) @douglowe ?